packages/docgen/scripts/dump-ast.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+const ts = require( 'typescript' );
+const { readFileSync } = require( 'fs' );
+const { inspect } = require( 'util' );
+
+const filename = process.argv[ 2 ];
+
+const sourceFile = ts.createSourceFile(
+	filename,
+	readFileSync( filename ).toString(),
+	ts.ScriptTarget.ES2020
```

Comment:
Should we use an evergreen target like `ESNext`?

```suggestion
	ts.ScriptTarget.ESNext
```